src/Rings/MPolyMap/MPolyQuo.jl

```diff
@@ -110,6 +110,15 @@ function _evaluate_plain(F::MPolyAnyMap{<: MPolyQuoRing}, u)
   return evaluate(lift(u), _images(F))
 end
 
+function _evaluate_plain(F::MPolyAnyMap{<:MPolyQuoRing, <:MPolyQuoRing}, u)
+  # This workaround deals with the fact that arithmetic in quotient rings is TERRIBLY slow.
+  # All the simplify calls make it unusable in this case, probably due to the fact that 
+  # setting `is_reduced` flags does not pay off in such iterative procedures. 
+  A = codomain(F)
+  v = evaluate(lift(u), lift.(_images(F)))
+  return A(v)
+end
```

Comment:
@thofma : Would this change be OK with you? We have an example which goes from ~14 seconds down to ~1.3 seconds only because of this.

Reply:
Makes sense

Reply:
@HereAround : Your blowups should run 10 times as fast now.

Reply:
Wonderful! Thank you!

Reply:
I am cleaning up this PR (hence, holding back a fair amount of changes locally right now, in favor of each commit individually for mistakes/errors). My finding is that this change in the quotient ring evaluation leads to changed printing of quotient ring maps (-> impact on book?) and thus failures in the doctests.

@HechtiDerLachs and @thofma, can you take a look and let me know what to do about them?

Reply:
Seems that the images of the generators are not reduced anymore. But I am not sure how this is effected by this change here. Would be good if @HechtiDerLachs could investigate.

Reply:
Shame on me - Matthias is on well-deserved vacation this week and the next(and so should not have pinged him).

Reply:
@thofma Maybe we can think of a fix? Otherwise I will move this change to a separate PR, so that I can finalise the changes here (even though they will be slower without the suggested quotient gymnastics) by the end of this week.

Thoughts?

Reply:
I don't have too much time right now, but maybe `return simplify(A(v))` does the trick? Or whatever is the function to simplify in the quotient.

Reply:
Ok. Let me try that.

Reply:
Great. This works. Thank you!